src/client.cr

```diff
@@ -3,6 +3,30 @@ require "db/pool"
 require "./connection"
 
 module Redis
+  # Set a global current client
+  #
+  # Example
+  # ```
+  # Redis.current = Redis::Client.new(URI.parse(ENV.fetch("REDIS_URL", "redis://127.0.0.1:6379/0")))
+  #
+  # # somewhere in the app
+  # def redis
+  #   Redis.current
+  # end
+  #
+  # # or just use a simple local scope
+  #
+  # redis = Redis.current
+  # ```
+  # > Note: Setting `Redis.current` can only happen once to avoid accidental override
+  def self.current=(client : Redis::Client)
+    @@current ||= client
+  end
```

Comment:
If this is already set, it will fail silently. I like the idea of avoiding the accidental override, but it might be worth failing more loudly to let people using this library know that their application is probably doing something they didn't expect.

Reply:
Great point! I like that approach much better, we can raise an exception to flag this on compile.